x/programs/cmd/simulator/cmd/config.go

```diff
@@ -33,6 +28,8 @@ const (
 	/// function call. A program's function can internally optionally call other
 	/// functions including program to program.
 	EndpointExecute Endpoint = "execute"
+	/// Create a new program on-chain
+	EndpointCreateProgram Endpoint = "createprogram"
```

Comment:
```suggestion
	CreateProgram Endpoint = "createprogram"
```

These names are a bit repetitive, their purpose is already written in the type